go.mod

```diff
@@ -7,7 +7,6 @@ require (
 	github.com/onsi/ginkgo/v2 v2.6.0
 	github.com/onsi/gomega v1.24.1
 	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
-	github.com/openshift/machine-config-operator v0.0.1-0.20200913004441-7eba765c69c9
```

Comment:
Could you please run `go.mod tidy`? 

It may be worth adding a lint for that WDTY?